docs/intro.md

```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_position: 1
+slug: /
```

Comment:
This makes the intro page the landing page for the docs site. Until we have some marketing materials for a true lander, I think it makes sense to have the docs intro be the landing page.